gui/mass-remove.lua

```diff
@@ -0,0 +1,314 @@
+--building/construction mass removal/suspension tool
+
+--[====[
+
+gui/mass-remove
+===============
+Allows removal of buildings/constructions and suspend/unsuspend using
+a box selection.
+
+The following marking modes are available.
+
+- Suspend (s): suspends the construction of a planned building/construction
```

Comment:
I started doing this then realized that there's no indication on the screen about whether a building is buildingplan or not. So a user could try to unsuspend a bunch of buildings and not know why it's not working, because they don't realize it's because those buildings are from buildingplan.

I tried a bunch of things to indicate the buildings, wanted to do something similar to the `pathable` plugin, which tints the screen based on conditions, but it seems like it does that through an actual C++ call, and trying to replicate that in Lua didn't work for me.

It's probably not that bad though, as long as the user knows that buildingplan buildings cannot be suspended/unsuspended, they should figure out why their suspend/unsuspend actions aren't actually being applied.

Reply:
Applied in 2896b67